src/entry_point/mod.rs

```diff
@@ -79,6 +79,10 @@ struct LayoutHeader {
     stack_size: usize,
 }
 
+//Procedural macros can't generate expressions so we generate a function
+//instead to parse thise. You will get a compile error if this isn't set!
+libtock_codegen::make_env!("APP_HEAP_SIZE");
```

Comment:
I think this comment is not required. You get the idea once you read the Rustdoc of `make_env`.

Reply:
I wanted to at least put something since it's not immediately obvious where the function comes from and I prefer to err on the side for more comments rather than fewer. Would there be something better to put here?